### PR TITLE
SRM-9:OA:ADD:Create GitHub Adapter

### PR DESCRIPTION
**Pull request by mim3009**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1 +1 @@
-# circleci
\ No newline at end of file
+# circleci test
```
Comment (mim3009) [inline README.md:1]: test

**Comment by IvankoRambo** [comment]
Wrong PR title - JIRA issue can't be retrieved

**Comment by IvankoRambo** [comment]
JIRA error : StatusCodeError: 404 - {"errorMessages":["Issue does not exist or you do not have permission to see it."],"errors":{}}

**Comment by IvankoRambo** [comment]
JIRA error : StatusCodeError: 404 - {"errorMessages":["Issue does not exist or you do not have permission to see it."],"errors":{}}

**Comment by IvankoRambo** [comment]
Wrong destination branch. Estimated branch is ```NO_ESTIMATED_BRANCH```

**Comment by IvankoRambo** [comment]
Wrong destination branch. Estimated branch is ```develop```

**Review by 308255 — APPROVED** [review]

**Review by 308255 — APPROVED** [review]

**Review by 308255 — APPROVED** [review]

**Review by 308255 — APPROVED** [review]

**Review by 308255 — APPROVED** [review]

**Review by 308255 — APPROVED** [review]

**Review by 308255 — APPROVED** [review]

**Pull request merged**